### PR TITLE
dev/v3.0/v2.1: add a tip for binlogctl's usage

### PR DESCRIPTION
**Pull request by juliezhang1112**
### What is changed, added or deleted? 
- Add a tip for binlogctl's usage

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1663

### Which version does your change affect? 
dev, v3.0, v2.1

### Checklist 

- [ ] Add a new file to `TOC.md`
- [ ] No Tab spaces anywhere 
- [ ] Leave a blank line both before and after a code block
- [ ] Keep the first level heading consistent with `title` in metadata
- [ ] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
LGTM

**Comment by juliezhang1112** [comment]
Please help review this PR, Thanks a lot! @TomShawn

File: v3.0/how-to/maintain/tidb-binlog.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ Command example:
     bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd update-pump -node-id ip-127-0-0-1:8250 -state paused
     ```
 
-    This command modifies the state of Pump or Drainer saved in the Placement Driver (PD).
+    This command modifies the state of Pump or Drainer saved in the Placement Driver (PD) and is only used if Pump or Drainer service fails to function properly.
```
Comment (TomShawn) [inline v3.0/how-to/maintain/tidb-binlog.md]: From the context, the reader must use this command very carefully. This information would be better understood if it is in the note format and active voice is used.
```suggestion
    > **Note:**
    >
    > This command modifies the state of Pump or Drainer saved in the Placement Driver (PD).
    > Use this command **only** when the Pump or Drainer service fails to function properly. 
```

**Comment by juliezhang1112** [comment]
PTAL :) @TomShawn

**Pull request merged**